static/gsApp/views/amCheckout/index.tsx

```diff
@@ -137,13 +138,18 @@ class AMCheckout extends Component<Props, State> {
     ) {
       props.onToggleLegacy(props.subscription.planTier);
     }
-    // TODO(checkout v3): remove these checks once checkout v3 is GA'd and we remove the legacy checkout route
+    // TODO(checkout v3): remove these checks once checkout v3 is GA'd and we've remove the legacy checkout route
```

Comment:
nit:
The `TODO` is misplaced, should be on the `if` condition instead of the `const query` declaration.

Reply:
good catch